src/shared/base-filesystem.c

```diff
@@ -78,6 +78,9 @@ static const BaseFilesystem table[] = {
 #elif defined(__riscv)
 #  if __riscv_xlen == 32
 #  elif __riscv_xlen == 64
+        { "lib64",    0, "usr/lib/riscv64-linux-gnu\0"
+                         "usr/lib64\0",                "ld-linux-riscv64-lp64d.so.1" },
```

Comment:
here too, the conditionalization on the existence of the ld.so doesn't look right given that it's in lib, not in lib64

Reply:
`/lib64/ld-linux-riscv64-lp64d.so.1 -> ld-2.32.so`

Reply:
but /lib64/ld-linux-riscv64-lp64d.so doesn't really matter, does it? i.e. ELF binaries will list /lib/ld-linux-riscv64-lp64d.so in their ELF headers as dynamic loader, notthing else, hence /lib64 doesn't really have to be there as a symlink

Reply:
mayba then add a comment here too to the sources: `/* same situation as for aarch64 */`